server/portal/apps/onboarding/steps/mfa.py

```diff
@@ -16,6 +16,8 @@ def display_name(self):
     def description(self):
         return """First, set up multi-factor authentication on the
                 <a style="color:#9d85ef;" href="https://portal.tacc.utexas.edu">TACC User Portal</a>
+                using 
```

Comment:
Maybe include `"the"` here, as in `"the TACC Token App"`
```suggestion
                using the
```

Reply:
I've updated the wording as suggested and pushed the changes to this branch.